apps/app-trade-agents/src/navigation/index.tsx

```diff
@@ -3,3 +3,4 @@ export { default as OrdersNavigator } from './Root/OrdersNavigator';
 export { default as DebetsNavigator } from './Root/DebetsNavigator';
 export { default as GoodMatrixNavigator } from './Root/GoodMatrixNavigator';
 export { default as MapNavigator } from './Root/Maps/MapNavigator';
+export { default as ShipmentNavigator } from './Root/ShipmentNavigator';
```

Comment:
The code patch looks good. The addition of the ShipmentNavigator seems to be a logical extension of the existing navigators. No bug risks or security vulnerabilities found.

Quality Rate: 8/10